.github/workflows/build_and_test.yml

```diff
@@ -87,6 +87,25 @@ jobs:
     uses: ./.github/workflows/build-build-tools-image.yml
     secrets: inherit
 
+  lint-openapi-spec:
```

Comment:
Could you please align it with `validate-compute-manifest` job, i.e. create a make command for that, add an `if`-condition, etc